docker-compose.yml

```diff
@@ -47,5 +47,8 @@ services:
       - "27017:27017"
   clearlydefined_mongo_seed:
     image: "clearlydefined/docker_dev_env_experiment_clearlydefined_mongo_seed"
+    build:
```

Comment:
Need to publish the (mongo_seed image)[https://hub.docker.com/r/clearlydefined/docker_dev_env_experiment_clearlydefined_mongo_seed] after this change